### ISSUE TITLE
Plugin does not work in docker terraform environment

### ISSUE THREAD

**Issue by insekticid** [hashicorp/terraform-provider-helm#104]
> Error: Error asking for user input: 1 error(s) occurred:
* provider.helm: fork/exec /root/.terraform.d/plugins/terraform-provider-helm: no such file or directory

Dockerized Terraform is based on golang:alpine linux v3.8

https://github.com/docker-library/golang/blob/870d14b02b4c226eb57beb5f44e1b22c12121641/1.11-rc/alpine3.8/Dockerfile

Dockerfile
 ```
FROM hashicorp/terraform:0.11.8

RUN apk -Uuv add ca-certificates openssl groff less git bash wget make jq curl unzip sed

WORKDIR /app

ENTRYPOINT ["/bin/terraform"]

CMD ["--help"]

ENV HELM_VERSION=v0.5.1
ENV HELM_FILENAME=terraform-provider-helm_${HELM_VERSION}_linux_amd64.tar.gz
ENV HELM_TERRAFORM_URL=https://github.com/mcuadros/terraform-provider-helm/releases/download/${HELM_VERSION}/${HELM_FILENAME}
ENV HELM_TERRAFORM_SHA256SUM=

RUN echo "Install Terraform plugin from:" \
  && echo "${HELM_TERRAFORM_URL}"
RUN mkdir -p ~/.terraform.d/plugins/
RUN wget -q ${HELM_TERRAFORM_URL}
RUN tar -xvf ${HELM_FILENAME}
RUN mv terraform-provider-helm*/terraform-provider-helm ~/.terraform.d/plugins/
RUN rm -f ${HELM_FILENAME}
 ```

Plugin works after recompile
```
RUN apk add --no-cache musl-dev go && \
    export PATH=$PATH:$(go env GOPATH)/bin && export GOPATH=$(go env GOPATH) && \
    mkdir -p $GOPATH/src/github.com/mcuadros && cd $GOPATH/src/github.com/mcuadros && \
    git clone https://github.com/mcuadros/terraform-provider-helm.git && \
    cd $GOPATH/src/github.com/mcuadros/terraform-provider-helm && \
    make build && \
    mv terraform-provider-helm ~/.terraform.d/plugins/ && \
    apk del musl-dev go
```

**Comment by smola** [comment]
Duplicate of https://github.com/mcuadros/terraform-provider-helm/issues/59

**Comment by meyskens** [comment]
Fixed in #111
